models/getComments.model.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../db/connection')
+
+function getAllComments (article_id) {
+    const queryString = `
+    SELECT comment_id, votes, created_at, author, body
+    FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC;
+    `
+    return db.query(queryString, [article_id])
+    .then(({ rows }) => {
+        return rows
+    })
+    .catch((err) => {
+        next(err)
+    })
```

Comment:
`next` does not exist in this function. You're nesting a catch block here which you don't need to do. The controller can catch any errors that occur on the end of the promise chain 👍